### PR TITLE
Fix #13434 Support setting conf path by argument

### LINKED ISSUE #13434: [Bug]: The default path of configDir
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Environment

```markdown
- Milvus version: 2.0.0-rc8
- Deployment mode(standalone or cluster): standalone
- OS(Ubuntu or CentOS): Ubuntu
```

### Current Behavior

I compiled Milvus from source and then i run milvus binary file on another computer.

If i don't set the environment variable `MILVUSCONF`, and there is no config file in the running path. The default configDir is the path where i complied Milvus.
 
截屏2021-12-15 下午3 25 35

### Expected Behavior

I want to set the default configDir to be a fixed path, not the absolute path where the Milvus compiled.

### Steps To Reproduce

_No response_

### Anything else?

_No response_

### PR DESCRIPTION
**Pull request by SpikeWong**
Signed-off-by: anoymouscoder <809532742@qq.com>
issue: #13434

### CODE REVIEW

**Comment by anoymouscoder** [comment]
issue: #13434

File: cmd/main_test.go [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+		assert.Equal(t, os.Getenv(metricsinfo.MilvusUsedGoVersion), GoVersion)
+		assert.Equal(t, os.Getenv(paramtable.MilvusConfEnvKey), milvusConf)
+	})
+	t.Run("milvusConf is empty ", func(t *testing.T) {
+		dummyPath := "path"
+		os.Setenv(paramtable.MilvusConfEnvKey, dummyPath)
+		injectVariablesToEnv()
+		assert.Equal(t, os.Getenv(paramtable.MilvusConfEnvKey), dummyPath)
+	})
+}
```
Comment (anoymouscoder) [inline cmd/main_test.go:28]: Check ci if pass while it failed in the local environment because of compatible issues

**Comment by haorenfsa** [comment]
> issue: #13434

add it in the body of this PR (in the first comment) , as below

```
Signed-off-by: anoymouscoder 809532742@qq.com
issue: #13434
```

File: go.mod [hunk 2]
```diff
@@ -55,7 +51,6 @@ require (
 
 replace (
 	github.com/apache/pulsar-client-go => github.com/apache/pulsar-client-go v0.6.1-0.20210728062540-29414db801a7 // BUGFIX #8803, update when pulsar-client-go has new release
-	github.com/dgrijalva/jwt-go => github.com/golang-jwt/jwt v3.2.2+incompatible // Fix security alert for jwt-go 3.2.0
```
Comment (haorenfsa) [inline go.mod:58]: Seems no need to change `go.mod` or `go.sum` files for this patch?

**Comment by haorenfsa** [comment]
@anoymouscoder CI pipeline failed. There're some new test cases, you can try `rebase` current `master`.

**Comment by anoymouscoder** [comment]
@haorenfsa done

**Comment by haorenfsa** [comment]
LGTM, owners take a look? @DragonDriver @cydrain

**Comment by haorenfsa** [comment]
@anoymouscoder the milvus members are now occupied with the milvus GA version, they would review this later, sorry for the delay.

**Comment by haorenfsa** [comment]
@anoymouscoder CI still failing, could you plz rebase the current master branch when you got time ?

**Comment by anoymouscoder** [comment]
> @anoymouscoder CI still failing, could you plz rebase the current master branch when you got time ?

done

**Comment by haorenfsa** [comment]
@jennyli-z jenkins CI status fail to update

**Comment by jennyli-z** [comment]
> @jennyli-z jenkins CI status fail to update

it is not used, you can ignore it for now, ci have passed ,and label `ci-passed` has already been added.

**Comment by haorenfsa** [comment]
> > @jennyli-z jenkins CI status fail to update
> 
> it is not used, you can ignore it for now, ci have passed ,and label `ci-passed` has already been added.

@xiaofan-luan seems ready to be reviewed & merged

**Comment by xiaofan-luan** [comment]
@anoymouscoder hi there, any progress on the pr? seems that the code need to rebase

**Pull request closed without merging**